jobs/common/pre-reqs.yaml

```diff
@@ -71,11 +71,10 @@
 
         # Install microk8s and kubectl
         sudo snap install microk8s --classic
-        sudo microk8s enable dns storage
         sudo snap install kubectl --classic
         echo "waiting for microk8s storage to become available"
         NEXT_WAIT_TIME=0
-        until [ $NEXT_WAIT_TIME -eq 30 ] || microk8s status --yaml | grep -q 'storage: enabled'; do
+        until [ $NEXT_WAIT_TIME -eq 30 ] || sudo microk8s enable dns hostpath-storage || microk8s status --yaml | grep -q 'storage: enabled'; do
```

Comment:
Is the logic here correct? This is saying "until timeout reached OR 'microk8s enable ...' succeeds OR 'microk8s status' includes 'storage: enabled'", I think. But that will terminate as soon as 'microk8s enable ...' succeeds, right? Do you want it to be `until A || ( B && C )` instead?

Reply:
ah yes, ty